webapp/src/main/java/com/box/l10n/mojito/service/security/user/session/MySQLDatabaseCondition.java

```diff
@@ -0,0 +1,21 @@
+package com.box.l10n.mojito.service.security.user.session;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.util.StringUtils;
+
+/**
+ * Conditional that checks if a MySQL database is in use by checking if the
+ * <b>spring.datasource.url</b> property is prefixed with <b>jdbc:mysql:</b>
+ *
+ * @author maallen
+ */
+public class MySQLDatabaseCondition implements Condition {
+
+    @Override
+    public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        String datasourceUrl = context.getEnvironment().getProperty("spring.datasource.url");
```

Comment:
could `DBUtils` be reused/merged in some ways? not super important but if we can avoid duplication and have a single place with that logic.

Reply:
I missed the DBUtils class, I don't think it can be merged into the condition as the condition logic is executed before the DBUtils bean is initialised but I could remove the conditional and autowire it into the CreateSessionAttributeInsertQuery bean. I'll update to execute the check in there and if there's not a mysql db then  we'll log a warning stating the custom query has not been set.